### PR TITLE
Oneclick prod testing

### PR DESCRIPTION
**Pull request by daryanypl**

### CODE REVIEW

File: bin/oneclick_library_import [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
 """Update the circulation manager server with new books from the
-OneClick content server."""
+OneClick content server.
+
+If you want to test with sample OneClick data on prod database, run like so:
```
Comment (aslagle) [inline bin/oneclick_library_import]: I don't think we'd ever want sample data on a prod database - do you mean a local database or a QA database?

Comment (daryanypl) [inline bin/oneclick_library_import]: Local, QA, or prod.  I had a situation where I wanted to make sure I was able to run a catalog populating script, and save the results to the database, and I didn't want to deal with the entire catalog.  I used the small sample catalog file that I created earlier for a unit test, and was able to troubleshoot my database connection and handling.

Comment (aslagle) [inline bin/oneclick_library_import]: That's fine, but I would take the word "prod" out of the comment. You definitely wouldn't want to run this with a production database that was actually in use.

Comment (daryanypl) [inline bin/oneclick_library_import]: done

**Comment by leonardr** [comment]
The substantive commit in here looks good.

**Pull request merged**